packages/editor/src/components/rich-text/index.native.js

```diff
@@ -94,6 +95,10 @@ export class RichText extends Component {
 		);
 	}
 
+	onEnter( event ) {
```

Comment:
You need to remove the parameter here, now that you update the onEnter interface on the RN-Aztec repo.